docs/guides/sdks/android-sdk-reference/android-sdk-reference-appsflyerlib.md

```diff
@@ -785,7 +852,10 @@ Register a validation listener for the `validateAndLogInAppPurchase` API.
 
 **Usage example**
 
-### sendAdRevenue
+### sendAdRevenue (LEGACY)
+<span class="annotation-deprecated">Deprecated in v6.15.0</span>  
+
+(Supported until SDK v.6.14.2 For versions including and above v6.15.0 use [`logAdRevenue`](#logadrevenue))
```

Comment:
@dori-af This version `v.6.14.2` needs to fixed as well

Reply:
Fixed

Reply:
Fixed